examples/timestamping/frontend/src/plugins/blockchain.js

```diff
@@ -3,88 +3,77 @@ import axios from 'axios'
 import * as proto from '../../proto/stubs.js'
 
 const PER_PAGE = 10
-const SERVICE_ID = 130
+const SERVICE_ID = 4
```

Comment:
Just want to remind that it would be good to get this ID via `/api/system/v1/services` endpoint.